assets/src/domain/eventEditor/services/apollo/queries/tickets/queries.ts

```diff
@@ -8,7 +8,10 @@ export const TICKET_ATTRIBUTES: any = gql`
 		description
 		endDate
 		isDefault
+		isExpired
 		isFree
+		isOnSale
+		isPending
 		isRequired
 		isSoldOut
```

Comment:
not completely sure how I feel about these flags because on one hand they can simplify things and avoid a lot of extra unnecessary processing in some places, but that's only if they are diligently updated based on the other data that defines them. The extra logic required to ensure that those flags are always updated whenever their underlying data changes obviously adds complexity and increases the chances of making things fragile.

I'm hoping that because of how the Apollo mutations are more centralized, we'll be able to stay on top of these kinds of things and maintain the data integrity of the system.

Reply:
Yes, those flags result can be derived/ computed on demand.

Reply:
For consistency, these fields should be added to GQL schema as well.